ruth/simulator/simulation.py

```diff
@@ -64,6 +64,7 @@ class SimSetting:
     travel_time_limit_perc: float = 0.0
     seed: InitVar = None
     speeds_path: str = None
+    disable_stuck_detection: bool = True
```

Comment:
You don't need to change it, it's a small thing, but I think that it's more natural to think about enabling a feature, vs disabling it. So using `enable_stuck_detection: bool = False` seems more natural to me.